flixel/animation/FlxAnimation.hx

```diff
@@ -228,7 +228,7 @@ class FlxAnimation extends FlxBaseAnimation
 				if (looped && curFrame == numFrames - 1)
 				{
 					curFrame = loopPoint;
-					parent.fireLoopCallback(name);
+					parent.onLoopComplete.dispatch(name);
```

Comment:
let's revert this since it's how all the others work

Reply:
reverted